src/porepy/numerics/mixed_dim/assembler.py

```diff
@@ -534,132 +538,132 @@ def _operate_on_edge_coupling(
             # Global block index associated with this edge variable
             ei: int = self.block_dof[(e, edge_var_key)]
 
-            # Get variable name and block index of the master variable.
-            master_vals: Tuple[str, str] = coupling.get(g_master, None)
-            if master_vals is None:
+            # Get variable name and block index of the primary variable.
+            primary_vals: Tuple[str, str] = coupling.get(g_primary, None)
+            if primary_vals is None:
                 # An empty identifying string will create no problems below.
-                master_var_key = ""
-                master_term_key = ""
-                # If the master variable index is None, this signifies that
-                # the master variable index is not active
+                primary_var_key = ""
+                primary_term_key = ""
+                # If the primary variable index is None, this signifies that
+                # the primary variable index is not active
                 mi = None
```

Comment:
"mi" should be pi or primary_ind or something. Same for "si"

Reply:
I agree. Maybe 'prim_i' and 'sec_i' would be more representative.

Reply:
Solved. See ecaefe0